server.js

```diff
@@ -32,9 +32,9 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 // Routing
-app.use(`/api/v${process.env.API_VERSION}/questions`, questionRouter);
+app.use(`/api/v${process.env.API_VERSION}/questions`, authenticate, questionRouter);
 app.use(`/api/v${process.env.API_VERSION}/auth`, authRouter);
```

Comment:
Please move authRouter to the top of router list and index to the bottom of the list